src/main/java/com/minecolonies/api/items/ModToolTypes.java

```diff
@@ -0,0 +1,201 @@
+package com.minecolonies.api.items;
+
+import com.minecolonies.api.compatibility.Compatibility;
+import com.minecolonies.api.items.registry.ToolTypeEntry;
+import com.minecolonies.api.util.ItemStackUtils;
+import com.minecolonies.api.util.constant.Constants;
+import com.minecolonies.api.util.constant.translation.ToolTranslationConstants;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.EquipmentSlot;
+import net.minecraft.world.item.*;
+import net.minecraftforge.common.ToolActions;
+import net.minecraftforge.registries.DeferredRegister;
+import net.minecraftforge.registries.RegistryObject;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Supplier;
+
+public class ModToolTypes {
```

Comment:
Javadoc and formatting (entire class)

Reply:
Done. Though the CheckStyle config fails to load for me.